src/azure-cli/azure/cli/command_modules/appconfig/_validators.py

```diff
@@ -126,6 +126,7 @@ def validate_filter_parameters(namespace):
         for item in namespace.filter_parameters:
             param_tuple = validate_filter_parameter(item)
             if param_tuple:
+                # pylint: disable=unbalanced-tuple-unpacking
                 param_name, param_value = param_tuple
```

Comment:
Why we add style check bypass but no code change?

Reply:
Pylint version was recently upgraded in CLI which started throwing this error. 

---
In reply to: [452476359](https://github.com/Azure/azure-cli/pull/14170#discussion_r452476359) [](ancestors = 452476359)